packages/manager/modules/manager-pci-common/package.json

```diff
@@ -37,7 +37,8 @@
     "@ovh-ux/manager-tailwind-config": "^0.2.1",
     "clsx": "2.1.1",
     "date-fns": "^3.6.0",
-    "lodash.isequal": "^4.5.0"
+    "lodash.isequal": "^4.5.0",
+    "@ovh-ux/manager-core-utils": "^0.3.0"
```

Comment:
it looks like @ovh-ux/manager-core-utils is a private package, this would be an issue for releasing pci-common

Reply:
@frenautvh 
Yep it's a private package but if we add it to dependencies, it will be bundled by rollup so the code will be embedded into the minified JS file